app/models/question.rb

```diff
@@ -0,0 +1,45 @@
+class Question < ApplicationRecord
+  include Swagger::Blocks
+
+  belongs_to :assessment
+  belongs_to :format
+  belongs_to :created_by, class_name: 'User'
+  has_many :solutions, inverse_of: :question
+  has_many :assets, as: :owner
+
+  # TODO add a validate method to format and use it to verify content is well-formed
```

Comment:
Does this mean to use the format specification to validate the user-submitted content adheres to the spec?  I don't think we can do that.

Reply:
yeah, that was the intent. I don't see why we can't do so, rswag figured out how, I intended to dig into their code to figure that out eventually.